### PR TITLE
Removes old containers from configuration

### PR DESCRIPTION
**Pull request by dipak-pawar**
Containers having version less than JBoss AS 7 are no longer supported.

### CODE REVIEW

**Comment by bartoszmajsak** [comment]
Great! Thank you. 

Since we are on containers, can you review the docs? I think we have a bit of confusing / outdated statements there, e.g.:

> For managed containers it might make sense to automatically download & install the container during the build to make installation on a build server more easy. This is only supported for AS7 at this moment. You can also type --<TAB> to auto-complete with the desired options.

File: README.asciidoc [hunk 1]
```diff
@@ -16,10 +16,11 @@ If you need to install an additional container profile, run command:
 
 `arquillian-add-container`
 
-In above command if selected container is supported by https://github.com/arquillian/arquillian-container-chameleon[chameleon] then it is configured using chameleon, otherwise it will add required dependency to setup container.
+In above command if selected container is supported by https://github.com/arquillian/arquillian-container-chameleon[chameleon] then it is configured using chameleon, otherwise it will add required container adapter to setup container.
 
-For managed containers it might make sense to automatically download & install the container during the build to make installation on a build server more easy. This is only supported for AS7 at this moment.
-You can also type --<TAB> to auto-complete with the desired options.
+If container is not supported by chameleon, you have to install container manually.
```
Comment (bartoszmajsak) [inline README.asciidoc:21]: Isn't it provided now by https://github.com/forge/arquillian-addon/pull/33?

Comment (dipak-pawar) [reply README.asciidoc:21]: yes it is provided by #33

Comment (bartoszmajsak) [reply README.asciidoc:21]: Then I think this documentation is misleading, isn't it? We don't have to download anything manually, right?

Comment (dipak-pawar) [reply README.asciidoc:21]: I'll update it as part of #33

Comment (bartoszmajsak) [reply README.asciidoc:21]: Ok, merged. Please rebase #33 and update docs :)

**Pull request merged**